src/esp/assets/ResourceManager.cpp

```diff
@@ -68,20 +68,20 @@ constexpr char ResourceManager::DEFAULT_LIGHTING_KEY[];
 constexpr char ResourceManager::DEFAULT_MATERIAL_KEY[];
 constexpr char ResourceManager::PER_VERTEX_OBJECT_ID_MATERIAL_KEY[];
 
-const std::map<PrimObjTypes, const char*> ResourceManager::PrimitiveNames3DMap =
-    {{PrimObjTypes::CAPSULE_SOLID, "capsule3DSolid"},
-     {PrimObjTypes::CAPSULE_WF, "capsule3DWireframe"},
-     {PrimObjTypes::CONE_SOLID, "coneSolid"},
-     {PrimObjTypes::CONE_WF, "coneWireframe"},
-     {PrimObjTypes::CUBE_SOLID, "cubeSolid"},
-     {PrimObjTypes::CUBE_WF, "cubeWireframe"},
-     {PrimObjTypes::CYLINDER_SOLID, "cylinderSolid"},
-     {PrimObjTypes::CYLINDER_WF, "cylinderWireframe"},
-     {PrimObjTypes::ICOSPHERE_SOLID, "icosphereSolid"},
-     {PrimObjTypes::ICOSPHERE_WF, "icosphereWireframe"},
-     {PrimObjTypes::UVSPHERE_SOLID, "uvSphereSolid"},
-     {PrimObjTypes::UVSPHERE_WF, "uvSphereWireframe"},
-     {PrimObjTypes::END_PRIM_OBJ_TYPES, "NONE DEFINED"}};
+const std::map<PrimObjTypes, const char*> ResourceManager::PrimNames3DMap = {
```

Comment:
In general, changing "primitive" to "prim" was to help save space while also being understandable, since "prim" is a common abbreviation for these kinds of geometric constructs in graphics.

Reply:
This will be obviated by migrating all handling to an external class, of course.